13.go

```diff
@@ -349,11 +349,11 @@ func (hs *serverHandshakeState) readClientFinished13(hasConfirmLock bool) error
 	}
 
 	// client authentication
-	if certMsg, ok := msg.(*certificateMsg13); ok {
```

Comment:
this is still needed. Server needs to respond with an error in case client sends a certificate, but server didn't requested it
In 4.4.2:
"
   The client MUST send a Certificate message if and only if the server
   has requested client authentication
"
Same point was already raised here:
https://github.com/cloudflare/tls-tris/pull/69#discussion_r174789298

Reply:
This will still cause an `alertUnexpectedMessage`. If the server didn't request a cert, but the client sent it, the next message will be parsed as a Finished message, which will fail if it's a Certificate message.